src/util.js

```diff
@@ -197,3 +191,58 @@ export const importantify = (string) =>
     string.replace(
         IMPORTANT_RE,
         (_, base) => base + " !important;");
+
+const getBrowserProperties = () => {
+    if (!getBrowserProperties.availableStyles) {
+        getBrowserProperties.availableStyles = {};
+        const styles = Object.keys(window.getComputedStyle(document.documentElement, ''));
+        for (let i = 0; i < styles.length; i++) {
+            const style = styles[i];
+            if (isNaN(Number(style))) {
+                getBrowserProperties.availableStyles[style] = style;
+            }
+        }
+    }
+    return getBrowserProperties.availableStyles;
+};
+
+const stylePrefixes = ['Moz', 'webkit', 'ms', 'O'];
+const getVendorPrefix = (property) => {
+    const validProperties = getBrowserProperties();
+
+    const validatedProp = validProperties[property];
+    if (validatedProp) {
+        return validatedProp;
+    }
+    const capitalProp = property[0].toUpperCase() + property.substr(1);
+    for (let i = 0; i < stylePrefixes.length; i++) {
+        const prefix = stylePrefixes[i];
+        const prefixedProperty = `${prefix}${capitalProp}`;
+        if (validProperties[prefixedProperty]) {
+      // learn which styles the browser likes
+            validProperties[validatedProp] = prefixedProperty;
```

Comment:
`validatedProp` is falsy here (most likely `undefined`). Don't you want `validProperties[property] == ...` instead?